Source/HelixToolkit.Wpf.SharpDX/Utilities/ObjReader.cs

```diff
@@ -184,6 +184,12 @@ public Object3DGroup Read(Stream s, ModelInfo info = default(ModelInfo))
                         break;
                     }
 
+                    while (line.EndsWith("\\")) 

```

Comment:
Sorry, could you elaborate on that a little? The while condition will evaluate to false if it's an empty or whitespace string unless I'm missing something.